lib/eco/export/jruby.py

```diff
@@ -435,7 +508,26 @@ def annotate_tree(self, src_file_name, log_file_name):
                                         def_filename,
                                         method.name,
                                         { method.name : method.is_mega },
-                                        JRubyMorphismLine)
+                                        JRubyCallGraph)
+                    # Eval strings interpreted at runtime.
+                    if def_eval_strings:
+                        self._annotate_text(def_lineno,
```

Comment:
These one-parameter-per-line function calls make reading code really difficult, as so little fits on a screen. Let's please reformat all of them so that there are multiple parameters per line. I tend to wrap to 100 chars per line, but there's also a good argument for 80 chars. Even with the latter, this can be reformatted to:
```python
                        self._annotate_text(def_lineno, def_filename,
                                            method.name,
                                            ("%s" % ", ".join(def_eval_strings)),
                                            JRubyEvalStrings)
```